### ISSUE TITLE
php-cs-filter not work on Mac

### ISSUE THREAD

**Issue by cyvoit** [Glavin001/atom-beautify#359]
Mac OSX: 10.10.3
atom Version: 0.201.0
atom-beautify version: 0.27.5
node version:v0.12.2

Please kindly have a look.

![screen shot 2015-05-22 at 16 12 32](https://cloud.githubusercontent.com/assets/8186665/7767150/c606a2f2-00a2-11e5-8785-f90086b7b3c2.png)

**Comment by visualcookie** [comment]
Same problem.

Log:
```
Error: spawn php-cs-fixer ENOENT
  at exports._errnoException (util.js:734:11)
  at Process.ChildProcess._handle.onexit (child_process.js:1035:32)
  at child_process.js:1127:20
  at process._tickCallback (node.js:357:13)
```

**Comment by cyvoit** [comment]
It seems I didn't install php-cs-filter. But after I installed it and relaunched atom, it still gets the error.
![screen shot 2015-05-22 at 17 11 54](https://cloud.githubusercontent.com/assets/8186665/7767563/1e6a969e-00a6-11e5-8cc1-018b2baf93b3.png)

**Comment by Glavin001** [comment]
This error message means that Atom (Node.js calling `spawn` child process) could not find the executable `php-cs-fixer`. 

1) Confirm that `php-cs-fixer` is properly installed. See https://github.com/FriendsOfPHP/PHP-CS-Fixer#globally-homebrew `brew install homebrew/php/php-cs-fixer`
2) Run `Atom Beautify - Help Debug Editor`.

**Comment by cyvoit** [comment]
I add `php-cs-fixer.phar` path to Atom Beautify package settings and the error is gone.
Thanks for your help.

**Comment by Glavin001** [comment]
Excellent to hear! I will be making this clearer to users in a future release. Thanks for your feedback.

**Comment by rthijssen** [comment]
Sorry to bring this closed issue back to life but this looks like the most relevant issue to comment on.

I installed `php-cs-fixer` via Homebrew and it's working fine via the command line. But when running `beautify` via Atom it doesn't beautify my file. I don't get an error, or anything in the development console. Are there any ideas what might be going wrong? Or how I could best debug this?

**Comment by Glavin001** [comment]
> I installed php-cs-fixer via Homebrew and it's working fine via the command line. But when running beautify via Atom it doesn't beautify my file.

My best recommendation would be what I said at https://github.com/Glavin001/atom-beautify/issues/359#issuecomment-104629941

> What you do need to do is install php-cs-fixer and potentially configure the path if you are having troubles. To configure, go to Atom Beautify package settings -> Language Config - PHP - CS Fixer Path -> enter the absolute path to your php-cs-fixer.phar file.

---

> I don't get an error, or anything in the development console. Are there any ideas what might be going wrong? Or how I could best debug this?

Running command `Atom Beautify - Help Debug Editor` would be the best way to debug this. Paste the results back here as a comment.

**Comment by rthijssen** [comment]
Thanks for your quick reply @Glavin001. Unfortunately updating the path isn't doing the trick.

I've slightly sanitised my debug log. Hope you still have all the info you need: https://gist.github.com/rthijssen/171ed07b6a23a5552841

**Comment by Glavin001** [comment]
@rthijssen looks like it actually did work!

> spawn done 0  Fixed all files in 0.028 seconds, 5.250 MB memory used

So then this would be a bug with php-cs-fixer, since that definitely does not look beautified.
